### DISCUSSION TITLE
Firefox Developer Edition

### DISCUSSION THREAD

**Discussion by Okami-byte** [elrumo/macOS_Big_Sur_icons_replacements#1487] [Icon request] [answered]
Hi! I would absolutely love it if someone could take this Firefox logo and change it so that the color scheme matches that of this Firefox Developer Edition! I have been using Firefox Developer Edition, and I believe the silhouette of this icon is so simplistic and beautiful. I would also welcome an inspired take if you would like to :)

Original
![image](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/49962728/73d76801-ff2f-4e38-8425-e10d9791ee94)

Color inspo
![image](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/49962728/46265c11-9c47-401a-9b29-a0049d26eca1)

**Comment by vikasbhatia** [comment, marked as answer]
How's this? Icons will be available to download [on my profile](https://macosicons.com/#/u/sidewinder) once they're approved.
<img width="256" alt="Firefox Developer Edition" src="https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/11446387/34c952ac-2d15-4e80-9ef7-ca7ebe97c5ee">
<img width="256" alt="Firefox Developer Edition 2" src="https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/11446387/a1cbc9de-9fcd-4707-94b5-90a05eaa68f5">

**Reply by Okami-byte** [reply]
THESE ARE SO GOOD! I love them! Thank you so much. Way better than I could have done!

**Reply by Okami-byte** [reply]
The gradient effect is sublime

**Reply by vikasbhatia** [reply]
Aw thank you! That makes me really happy. I enjoy making them, so if you have any other requests in the future I'd be happy to do them. Have a good one!

**Reply by Okami-byte** [reply]
Will do! You've opened the flood gates now, lol

**Reply by Okami-byte** [reply]
Hey @vikasbhatia ,

Do you happen to know typical turn around time for the icons to be approved? Also, I wanted to know if it would be possible for you to slide me one of the icons to use in the meantime :)

**Reply by vikasbhatia** [reply]
@moonlinx It could be anywhere from a day to several weeks. As far as I know the site is just run by one dude and the review process is manual, so it's whenever he finds the free time to do it I guess.

But you can still start using the icons.

1. Right-click and download the PNGs I posted.
2. In your Applications folder, open the Get Info window of the app (right-click > Get Info or select it and ⌘+I)
3. Drag and drop the PNG onto the current icon.

See [this](https://macosicons.com/#/how-to) for a visual guide.

The ICNS that you get when you download an approved icon from the site is the "official" file format for Mac icons but the PNG works just fine too.

PS. Personally I use [Pictogram](https://pictogramapp.com) to change app icons. It makes changing icons a lot easier and it persists across app updates. If you change it manually with the steps above, it may get overwritten in an update.

**Reply by Okami-byte** [reply]
WHAT?!?! OH MY GOSH! THANK YOU!
